src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.CodeAnalysis.CommandLine
     ///  Field Name         Type                Size (bytes)
     /// ----------------------------------------------------
     ///  Length             Integer             4
-    ///  ProtocolVersion    Integer             4
+    ///  RequestId          Guid                16
```

Comment:
In the early days of the compiler server the `ProtocolVersion` field was intended to be a versioning mechanism. Every time we changed the protocol we update this number and make sure that we maintain compatibility. Later on though we added the compiler hash to the build protocol and both sides reject messages where the hashes don't match. This effectively renders the `ProtocolVersion` field meaningless hence I've removed it as a part of adding `RequestId` in.